src/View/Widgets/DiskRenderer.vala

```diff
@@ -69,8 +69,19 @@ public class Marlin.CellRendererDisk : Gtk.CellRendererText {
         context.add_class (Gtk.STYLE_CLASS_FRAME);
         context.render_background (cr, x, y, total_width, BAR_HEIGHT);
         context.render_frame (cr, x, y, total_width, BAR_HEIGHT);
+
         /* Filled part of bar */
-        context.add_class ("fill-block");
+        double filled_percent = ((double) disk_size - (double) free_space) / (double) disk_size;
+        if (filled_percent >= 0.9) {
+            context.add_class ("fill-block-critical");
+        }
+        else if (filled_percent >= 0.75) {
+            context.add_class ("fill-block-warn");
+        }
+        else {
```

Comment:
Elementary code style: else follows brace on same line.   `} else if () {`

Reply:
> @CodingSpiderFox Thanks for your first contribution! I agree that the colors are more distinct with a slightly thicker bar and I am happy for the bar height to increase to 4px. I find 5px a little too thick for my liking. I think any more change to the appearance would require prior approval of the UX team so perhaps would be best raised as a separate issue/PR where details of colors etc can be fully debated.
> 
> There is a minor comment inline re elementary code style

So should I add the 4px to this PR or open another?

Reply:
I am happy for it to be included here.

Reply:
Done